tests/assertions.js

```diff
@@ -1,9 +1,7 @@
 /*!
- * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
```

Comment:
technically speaking this should be `2021-2022`, but this is really not worth holding up a PR over.